### DISCUSSION TITLE
Resigning maintainer

### DISCUSSION THREAD

**Discussion by MortenDHansen** [owen-it/laravel-auditing#925] [Announcements]
Due to job change I am leaving the PHP ecosystem for a while and will no longer be able to help in this project. Thanks for your help, and I wish you all and the project the best going forward.

**Comment by parallels999** [comment]
😭😭😭 it worries me a lot

**Comment by MitchTalmadge** [comment]
Totally understand and wish you well!

**Comment by jose123v** [comment]
**CC:** @anteriovieira, @quetzyg

**Comment by quetzyg** [comment]
> **CC:** @anteriovieira, @quetzyg

Mate, I haven't been a contributor/maintainer for years.

**Comment by willpower232** [comment]
Regular user of this package here, thanks @MortenDHansen for all the releases, Is @anteriovieira the only maintainer left or is anyone else taking over?

**Reply by anteriovieira** [reply]
Hi @willpower232 , I'm no longer working on projects with Laravel, which makes it difficult for me to get in touch with Laravel Auditing.

**Comment by anteriovieira** [comment]
Hi everyone, I appreciate the efforts of all contributors and the entire community to keep the package active. At the moment I don't have free time to continue. 

I'm looking for someone who can help me with this, I found one but they also ran out of time now. 

If anyone is interested, please contact me, any help is welcome.

**Reply by erikn69** [reply]
I was able to help with basic tasks such as keeping new Laravel/dependencies versions updated, or fix bugs
but in the case of features/ new major releases,  I still don't have much confidence to myself

**Reply by willpower232** [reply]
We use this package as a standard package in our dozens of Laravel projects and it has definitely saved our bacon on several occasions in the past few years so happy to help out if that helps you keep it going.

We already keep a few packages going ourselves https://github.com/orgs/SynergiTech/repositories?q=lang%3Aphp&type=public

**Reply by erikn69** [reply]
>it has definitely saved our bacon on several occasions in the past few years

me too, having audits has been my best defense, or even to know what happened and decide how to proceed. 😄

In any case my contributions will remain the same.
https://github.com/owen-it/laravel-auditing/commits?author=erikn69

**Reply by PaolaRuby** [reply]
https://github.com/owen-it/laravel-auditing/graphs/contributors

https://github.com/owen-it/laravel-auditing/commits?author=erikn69
https://github.com/owen-it/laravel-auditing/commits?author=willpower232

**Reply by willpower232** [reply]
Not sure what you are trying to say, I haven't needed to contribute so far but I have a vested interest in keeping this package alive and am happy to help out however is needed 😅

**Reply by PaolaRuby** [reply]
> I haven't needed to contribute so far but I have a vested interest in keeping this package alive and am happy to help out however is needed 

Me too, Maybe that's reason enough for me to be a maintainer. 😄🚀✨


How about any of the existing members?
https://github.com/orgs/owen-it/people

**Reply by anteriovieira** [reply]
Hey guys, please let me know your discord nickname so I can add you as maintainers.

@parallels999 @erikn69 @willpower232 @PaolaRuby

**Reply by erikn69** [reply]
ensystems

**Reply by niladam** [reply]
Hello

I'd like to help as well if still possible. 

My discord username is emton13.

**Reply by parallels999** [reply]
@niladam, you can help by contributing with PR, I will review them as soon as possible.

**Comment by CCH4GITHUB** [comment]
I know the auditable_id  must be an integer.
What can I do if the primary key must be a varchar

**Reply by parallels999** [reply]
Something like #992 but for auditable
